src/IO/H5/Python/File.cpp

```diff
@@ -49,8 +50,22 @@ void bind_h5file() {
            +[](const h5::H5File<h5::AccessType::ReadWrite>& f) {
              f.close_current_object();
            })
-      .def("groups", +[](h5::H5File<h5::AccessType::ReadWrite>& f) {
-        return std_vector_to_py_list<std::string>(f.groups());
+      .def("groups",
+           +[](h5::H5File<h5::AccessType::ReadWrite>& f) {
+             return std_vector_to_py_list<std::string>(f.groups());
+           })
+
+      .def("get_vol",
+           +[](const h5::H5File<h5::AccessType::ReadWrite>& f,
+               const std::string path) {
```

Comment:
Sorry I missed this last time, but the strings should be taken by `const std::string&` (reference at the end). Below as well in `insert_vol`